### PR TITLE
improve(api): Standardize relayer fee calculations from sdk-v2

### PR DESCRIPTION
**Pull request by nicholaspai**
- [This PR](https://github.com/across-protocol/sdk-v2/pull/145) improves SDK-v2 so that it exports `isAmountTooLow` and `minDeposit` which means that /limits and /suggested-fees can compute those values using the same math
- Independently, we should also set the capital costs percent from an env var in the same way that the FE does

Resolves ACX-480

### CODE REVIEW

File: api/_utils.ts [hunk 1]
```diff
@@ -38,6 +39,10 @@ export const DEFAULT_GAS_MARKUP = 0;
 
 export const HUP_POOL_CHAIN_ID = Number(REACT_APP_HUBPOOL_CHAINID || 1);
 
+export const FLAT_RELAY_CAPITAL_FEE = REACT_APP_FLAT_RELAY_CAPITAL_FEE
+  ? Number(process.env.REACT_APP_FLAT_RELAY_CAPITAL_FEE)
+  : 0;
```
Comment (pxrl) [inline api/_utils.ts:44]:
```suggestion
export const FLAT_RELAY_CAPITAL_FEE = Number(REACT_APP_FLAT_RELAY_CAPITAL_FEE ?? 0);
```

Is there a valid case for ever having a 0% relayer capital fee? How about we stick to something like 3bps and then override via REACT_APP_FLAT_RELAY_CAPITAL_FEE?

File: api/_utils.ts [hunk 2]
```diff
@@ -360,10 +365,14 @@ export const getRelayerFeeCalculator = (destinationChainId: number) => {
...
   const relayerFeeCalculatorConfig = {
     feeLimitPercent: maxRelayFeePct * 100,
-    capitalCostsPercent: 0.04,
+    capitalCostsPercent: FLAT_RELAY_CAPITAL_FEE, // This is set same way in ./src/utils/bridge.ts
     queries: queryFn(),
     capitalCostsConfig: relayerFeeCapitalCostConfig,
   };
+  if (relayerFeeCalculatorConfig.feeLimitPercent < 1)
+    throw new Error(
+      "Setting fee limit % too low will produce nonsensical relay fee details"
```
Comment (pxrl) [inline api/_utils.ts:374]: Consider specifying the actual limit as part of the exception?

File: api/_utils.ts (hunk 1 above)
Comment (nicholaspai) [reply api/_utils.ts:44]: i like that

**Review by pxrl — APPROVED** [review]
No major comments; only some suggestions around nits, for consideration.

**Review by pxrl — APPROVED** [review]
LGTM!

**Review by pxrl — APPROVED** [review]

File: api/_utils.ts [hunk 3]
```diff
@@ -37,11 +36,11 @@ export const gasMarkup = GAS_MARKUP ? JSON.parse(GAS_MARKUP) : {};
 // Default to no markup.
 export const DEFAULT_GAS_MARKUP = 0;
 
-export const HUP_POOL_CHAIN_ID = Number(REACT_APP_HUBPOOL_CHAINID || 1);
+export const HUP_POOL_CHAIN_ID = Number(REACT_APP_HUBPOOL_CHAINID ?? 1);
```
Comment (pxrl) [inline api/_utils.ts:39]: One observation here - this change permits a chain ID of 0:

```
> 0 || 1
1
> 0 ?? 1
0
```

Comment (nicholaspai) [reply api/_utils.ts:39]: ah i see ill revert

File: api/_utils.ts [hunk 4]
```diff
@@ -36,8 +36,14 @@ export const gasMarkup = GAS_MARKUP ? JSON.parse(GAS_MARKUP) : {};
 // Default to no markup.
 export const DEFAULT_GAS_MARKUP = 0;
 
+// Don't permit REACT_APP_FLAT_RELAY_CAPITAL_FEE=0
 export const HUP_POOL_CHAIN_ID = Number(REACT_APP_HUBPOOL_CHAINID || 1);
```
[the remark this answers is not in the document]
Comment (nicholaspai) [reply api/_utils.ts:40]: no its ok to have capital fee be 0 but 0 as a default doesn't make the most sense

**Pull request merged**